reaper/task.go

```diff
@@ -225,25 +261,36 @@ func (r *GroupReaper) Run() {
 				r.delete(m)
 			}
 		case task.Ready:
-			if len(m.Tasks) == 0 {
-				r.delete(m)
-				continue
-			}
-			if m.HasBucket() {
+			mark := m.CreateTime
+			if time.Since(mark) > time.Hour {
 				r.release(m)
+				if len(m.Tasks) == 0 {
+					r.delete(m)
+				} else {
+					r.release(m)
+				}
 			}
 		}
 	}
 }
 
 // release resources.
 func (r *GroupReaper) release(m *model.TaskGroup) {
-	m.SetBucket(nil)
-	err := r.DB.Save(m).Error
-	if err == nil {
+	nChanged := 0
+	if m.HasBucket() {
 		Log.Info("Group bucket released.", "id", m.ID)
-	} else {
-		Log.Error(err, "")
+		m.SetBucket(nil)
+		nChanged++
+	}
+	if len(m.List) > 0 {
+		m.List = nil
```

Comment:
hint: Clear the list to reduce storage. After the group has been submitted, the tasks in the List have been created as _real_ tasks.